controllers/dashboard.go

```diff
@@ -1,19 +1,45 @@
 package controllers
 
 import (
+	"google-scraper/models"
+
+	"github.com/beego/beego/v2/core/logs"
 	"github.com/beego/beego/v2/server/web"
+	"github.com/beego/beego/v2/server/web/pagination"
 )
 
+var sizePerPage int
+
 type Dashboard struct {
 	baseController
 }
 
+func init() {
+	var err error
+	sizePerPage, err = web.AppConfig.Int("sizePerPage")
+	if err != nil {
+		logs.Error("Retrieving sizePerPage failed: ", err)
+	}
+}
+
 func (c *Dashboard) New() {
 	web.ReadFromRequest(&c.Controller)
+	keywords := models.GetKeywords(c.CurrentUser)
+	keywordsCount, err := keywords.Count()
+	if err != nil {
+		logs.Error("Retrieving keyword count failed: ", err)
+	}
+
+	paginator := pagination.SetPaginator(c.Ctx, sizePerPage, keywordsCount)
+	paginatedKeywords, err := models.GetPaginatedKeywords(c.CurrentUser, paginator.Offset(), sizePerPage)
+	if err != nil {
+		logs.Error("Retrieving keywords failed: ", err)
+	}
```

Comment:
I think this is business logic and it should not belong in the controllers. The controller should have only the HTTP handler. How about moving this to something like `services`?